src/components/StatusStrip/StatusStrip.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react";
+import { pluralize } from "app/utils";
+
+import "./_status-strip.scss";
+
+export default function StatusStrip({ statusList }) {
+  const [status, setStatus] = useState({});
+
+  useEffect(() => {
+    let proposedStatus = {};
+    let count = 0;
+
+    function incrementCount(incrementBy) {
+      count = count + incrementBy;
+    }
+
+    for (const [groupName, statuses] of Object.entries(statusList)) {
+      const statement = Object.values(statuses).map((status) => {
+        incrementCount(status.count);
+        return (
+          <span
+            key={status.label}
+            className={`status-icon is-${status.label.toLowerCase()}`}
+          >{`${status.label}: ${status.count}`}</span>
```

Comment:
The count goes in front of the status and the semicolon is removed:

Design:
<img width="391" alt="Screen Shot 2020-11-25 at 8 57 59 PM" src="https://user-images.githubusercontent.com/532033/100302935-fcd33c80-2f60-11eb-9590-3b6527c33ce4.png">

Implementation:
<img width="367" alt="Screen Shot 2020-11-25 at 8 58 50 PM" src="https://user-images.githubusercontent.com/532033/100302971-0eb4df80-2f61-11eb-8cb3-a1f5209d6b85.png">